lib/template.html

```diff
@@ -3,6 +3,8 @@
   <title>{{ seo_tag.title }}</title>
 {% endif %}
 
+<meta name="generator" content="Jekyll v{{jekyll.version}}" />
```

Comment:
Should have whitespace around the `{{`s to remain consistent.

Reply:
Based on the CI output, it may not actually be outputting the version.

Reply:
```html
<meta name="generator" content="{{ jekyll.version | prepend:'Jekyll ' }}" />
```

?

Reply:
You're right it doesn't.  So should it be included in the drop instead?

Reply:
Good catch! `jekyll` needs to be added to the payload.

https://github.com/jekyll/jekyll-seo-tag/blob/1b66cb1522d695c59b8753e6cdeaa6e65c16f9ca/lib/jekyll-seo-tag.rb#L46-L53

Reply:
@pathawks Same result. It looks like in the template it does not compute `jekyll.version`

Reply:
It looks like https://github.com/jekyll/jekyll/blob/master/lib/jekyll/drops/jekyll_drop.rb will need to be added to the payload @pathawks linked to. Out of scope for this PR, but it may make more sense to start with the `site_payload` (unified payload) drop and merge in the page and our drop rather than adding assumed globals one-off.

Reply:
As I have no idea how to deal with the `site_payload` and the `jekyll_drop` for now, I followed the current pattern used for other values and added a test and to my utter surprise it works. 💧